### ISSUE TITLE
HTTP 400 error message explanations: HTTP header vs. body

### ISSUE THREAD

**Issue by mmd-osm** [zerebubuth/openstreetmap-cgimap#124]
Apologies, if this has been reported before.

I'm wondering a bit, why the error message `Error: You requested too many nodes (limit is 50000). Either request a smaller area, or use planet.osm` is sent back as HTTP header field "Error" (which appears to be somewhat non-standard).

I'd rather expect this kind of information as part of the HTTP body. As a nice side effect, the error message would also be shown in the browser.

```
*   Trying 2001:630:12:500:219:bbff:fe39:8aba...
*   Trying 193.63.75.99...
* Connected to api.openstreetmap.org (2001:630:12:500:219:bbff:fe39:8aba) port 80 (#0)
> GET /api/0.6/map?bbox=1.9885,48.7659,2.5107,48.9755 HTTP/1.1
> Host: api.openstreetmap.org
> Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
> Accept-Encoding: gzip, deflate
> Accept-Language: de,en-US;q=0.7,en;q=0.3
> Connection: keep-alive
> DNT: 1
> Referer: http://www.openstreetmap.org/export
> Upgrade-Insecure-Requests: 1
> User-Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:50.0) Gecko/20100101 Firefox/50.0
> 
< HTTP/1.1 400 Bad Request
< Date: Sat, 24 Dec 2016 11:45:30 GMT
< Server: Apache/2.4.18 (Ubuntu)
< Content-Disposition: attachment; filename="map.osm"
< Error: You requested too many nodes (limit is 50000). Either request a smaller area, or use planet.osm
< Cache-Control: no-cache
< Content-Length: 0
< Content-Type: text/html; charset=utf-8
< Connection: close
< 
* Closing connection 0
```

**Comment by zerebubuth** [comment]
Thanks for pointing that out!

That's a regression and a bug. I've made an attempt at a fix in #125.

**Comment by mmd-osm** [comment]
Great, thanks a lot! Is there anything else left to do or can this issue be closed then?

**Comment by zerebubuth** [comment]
Yup, good point! Fixed in #125.
